### PR TITLE
fix(helm): fix setting home via `helm --home=HOME`

### PR DESCRIPTION
**Pull request by adamreese**

### CODE REVIEW

**Review by technosophos — APPROVED** [review]

**Review by technosophos — COMMENTED** [review]
Oh! I see... this is a potentially bigger issue. Ping me when you want me to re-review this

**Comment by technosophos** [comment]
@adamreese What's the status of this one?

File: cmd/helm/serve.go [hunk 1]
```diff
@@ -55,19 +55,29 @@ func newServeCmd(out io.Writer) *cobra.Command {
...
+			return srv.complete(args)
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return srv.run()
 		},
 	}
 
 	f := cmd.Flags()
-	f.StringVar(&srv.repoPath, "repo-path", settings.Home.LocalRepository(), "local directory path from which to serve charts")
+	f.StringVar(&srv.repoPath, "repo-path", "", "local directory path from which to serve charts")
```
Comment (adamreese) [inline cmd/helm/serve.go]: Working on a better fix for setting the default path here, but want to fix the bigger bug first.

Comment (technosophos) [inline cmd/helm/serve.go]: Okay, this works for me.

**Review by technosophos — APPROVED** [review]

**Pull request merged**